### ISSUE TITLE
Define more clearly what is an input gesture in the user guide

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#11915]
### Steps to reproduce:

* Put yourself in the position of a beginner with NVDA
* Open the user guide and try to find what is an input gesture

### Actual behavior:

"Input gesture" is an expression used in the user guide and in NVDA's interface, but a new user will have difficulty to know what it is. Even people with a basic usage of NVDA and no specific formation to it (e.g. coming from Jaws) may have difficulty to know what is exactly an input gesture.
This difficulty has become more important now that NVDA supports touch gestures, since the users may associate "gestures" with touch gestures only.
For example, on a French mailing list, a user has just written that he was using NVDA already for a while, but he just had discovered that "input gestures" were also "keyboard shortcuts" and not only "touch gestures".

### Expected behavior:

Have the concept of "input gesture" documented better and earlier in the user guide. Maybe in a subparagraph of the following section:
`4. Getting started with NVDA`

I would not recommend however to replace the "input gesture" terminology. The majority of NVDA users are used to it and I have no better terminology suggestion for it.

#### NVDA installed/portable/running from source:
any
#### NVDA version:
2020.3
#### Windows version:
any
#### Name and version of other software in use when reproducing the issue:
n/a
#### Does the issue still occur after restarting your computer?
n/a
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes. idem
#### If addons are disabled, is your problem still occuring?
n/a
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
n/a

**Comment by zstanecic** [comment]
Hi,

That is not even a problem just for user, but also for the translator.

Translator knows how to translate it in the target language, but the problem comes in when user starts to ask you, what this thing is.

**Comment by CyrilleB79** [comment]
> Hi, That is not even a problem just for user, but also for the translator. Translator knows how to translate it in the target language, but the problem comes in when user starts to ask you, what this thing is.

Yes, indeed. This was actually one of the reasons why I have opened this issue, since the user was also wondering if it was a translation issue.

**Comment by Qchristensen** [comment]
Looking up Input Gestures in the user guide (Section 12.2.3): https://www.nvaccess.org/files/nvda/documentation/userGuide.html#InputGestures

The first sentence states:

"In this dialog, you can customize the input gestures (keys on the keyboard, buttons on a braille display, etc.) for NVDA commands."

That seems pretty clear to me that it covers keyboard actions - in fact, where many assume it means touch gestures (which it does include), THOSE aren't specifically mentioned here.

Is the problem the definition, or the name of the feature itself?  I know @zstanecic mentioned to keep the term "Input gestures", but would changing that to say "Input gestures / shortcut keys" clear up the confusion?

I'm not overly in favour of adding it to the "Getting started" section, since ideally it's not something you necessarily want to change when you are new to NVDA.

**Comment by zstanecic** [comment]
Hi reef,

I propose the following:

Add the definitione which explains the term broadly to the user guide.

This definition should sound like that.

When it comes to NVDA, imput gesture has the following meanings:

*	Keyboard shortcut,
*	- gesture on the touchscreen,
*	- braille display keyboard shortcut.

I thing that it will be understandable, but better suggestions are welcome.

**Comment by Qchristensen** [comment]
@zstanecic isn't your proposal just taking the existing first sentence I quoted, and turning it into bullet points?  If that makes it easier that's fine, but the only new information there is the inclusion that input gestures also includes touch gestures - which I mentioned in my comment.

Also - @CyrilleB79 (in the original issue description) mentioned that this may be confusing for users coming from Jaws - what does Jaws call its equivalent feature?  The Switching from Jaws to NVDA guide only mentions Input Gestures once: https://github.com/nvaccess/nvda-community/wiki/SwitchingFromJawsToNVDA in the context of screen shade / screen curtain.  It would be useful to mention input gestures in the guide as well in the context of the name of the Jaws command as well.

**Comment by zstanecic** [comment]
Hi Quentin,

Turning that into bullet points with detailed explanation will make it more clear for the user.

**Comment by CyrilleB79** [comment]
> Also - @CyrilleB79 (in the original issue description) mentioned that this may be confusing for users coming from Jaws - what does Jaws call its equivalent feature? The Switching from Jaws to NVDA guide only mentions Input Gestures once: https://github.com/nvaccess/nvda-community/wiki/SwitchingFromJawsToNVDA in the context of screen shade / screen curtain. It would be useful to mention input gestures in the guide as well in the context of the name of the Jaws command as well.

I have had a quick look at Jaws interface and help. The equivalent of "Input gesture manager" dialog would be "Keyboard Manager" dialog and "Navigation Quick Key Manager". There does not seem to be a way to reassign touch gestures on Jaws. Thus, Jaws uses exclusively the word "gesture" in the context of touch gestures, hence probably the confusion of some people.

However, I have not been using Jaws regularly for a few years and I did never use a touch screen on a computer, either with Jaws or with NVDA. So anyone having more experience than me, feel free to correct me.

On the contrary to what I wrote before, maybe renaming the "Input gesture manager" would be a solution, keeping the word gesture only for touch gestures. What about "Command manager"?
Before to proceed however, renaming this dialog should be discussed widely and not only by 2 or 3 people commenting this issue. This would have an impact on the existing documentation, maybe training material, and established usage.

**Comment by Mike20001123** [comment]
As we know, NVDA uses the word "Commands" in "Commands Quick Reference" to refer to items found in "Input Gesture", I personally agree with CyrilleB79's idea of using "Command Manager" to replace "Input gestures" for unifying and simplifying terms in NVDA user interface. 
However, this replacement does affect user guides and training materials in all languages, we probably can design a survey being translated to all available languages asking for the best options. 
It is truly a dilemma for translators: I can either keep the terms as original which causes confusions for new users or translate those terms incorrectly to fit local user habits.

**Comment by Qchristensen** [comment]
If a change in the terminology would better help users, and particularly new users to become familiar with NVDA and understand features, then I think it is worth making.  I would guess that either translators have already used the most appropriate word / phrase in their language, or would have the option to if they are starting with a more appropriate English word / phrase.

The question then becomes, is "Command manager" a better term to start with than "Input gestures"?

**Comment by XLTechie** [comment]
On Thu, 25 Feb 2021, Quentin Christensen wrote:

> The question then becomes, is "Command manager" a better term to start with than "Input gestures"?

By far, IMO.

Honestly, in every other context, gestures are touch related.  No other software 
I know of thinks of gestures as involving the keyboard.

So to my mind, transitioning to anything more reasonable than gestures is a step 
in the right direction.  I have no problem with "commands".

**Comment by CyrilleB79** [comment]
@Adriani90 you have closed this issue only based on your very personal opinion. Though no one else who has written here seems to share it.

@zstanecic has acknowledged the issue with the current term (input gesture). @Qchristensen is open to the change if it can help. And @XLTechie and @ChillChenMike also agree to change to something like "Command manager".

It's not a matter of modernity and language evolution. It's a matter of confusion between general "input gestures" and specific "touch gestures".

Sorry, this issue doesn't really need to be closed this way. Reopening.

**Comment by ruifontes** [comment]
In portuguese we use "comandos", or in english "commands" and not "input gestures"...
So, I agree we should do something...
Explaining what is an "Input gesture" or completly replace "Input gestures" by "Commands".
